concent_api/core/tests/test_core_views.py

```diff
@@ -190,17 +190,22 @@ def test_send_should_return_http_400_if_data_is_incorrect(self):
         )
         self._test_400_response(response)
 
-        data = message.concents.ForceReportComputedTask()
-        data.report_computed_task = message.tasks.ReportComputedTask()
-        compute_task_def['deadline'] = self.message_timestamp - 3600
-        data.report_computed_task.task_to_compute = message.TaskToCompute(compute_task_def=compute_task_def)
-
-        response = self.send_request(
-            url='core:send',
-            data=dump(
+        data = message.concents.ForceReportComputedTask(
+            report_computed_task=message.tasks.ReportComputedTask(
+                task_to_compute=message.tasks.TaskToCompute(
+                    compute_task_def=message.tasks.ComputeTaskDef(),
+                    want_to_compute_task=message.tasks.WantToComputeTask()
+                )
+            )
+        )
+        malformed_dumped_message = dump(
                 data,
                 self.PROVIDER_PRIVATE_KEY,
-                CONCENT_PUBLIC_KEY),
+                CONCENT_PUBLIC_KEY,
+        )
+        response = self.send_request(
+            url='core:send',
+            data=malformed_dumped_message[:-1],
```

Comment:
please move this slicing (`[-:1]` to the variable definition - otherwise the name is wrong ;)

Reply:
Changed :)